llvm_mode/afl-clang-fast.c

```diff
@@ -286,6 +288,7 @@ static void edit_params(u32 argc, char** argv) {
       cc_params[cc_par_cnt++] = "none";
     }
 
+#ifndef __ANDROID__
```

Comment:
Why is this disabled on android?

Reply:
this I dont understand either

Reply:
Is https://github.com/google/AFL/pull/59/files/75d6ee4838a902178e232a592dd4d0b8356f5de5#r359633493 a response to this?